<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TurBoCapture</class>
 <widget class="QMainWindow" name="TurBoCapture">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TurBoCapture</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="horizontalSpinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>580</y>
      <width>74</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>630</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="verticalSpinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>620</y>
      <width>74</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>611</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="horizontalLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal</string>
    </property>
   </widget>
   <widget class="QLabel" name="verticalLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertical</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takeFrameButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>560</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Frame</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>221</width>
      <height>491</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="compileButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>640</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Compile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newRowButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>570</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>New Row</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="verticalSurfaceSpinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>620</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="horizontalSurfaceSpinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>580</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="cropLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>560</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop</string>
    </property>
   </widget>
   <widget class="QLabel" name="surfaceLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>560</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Surface</string>
    </property>
   </widget>
   <widget class="QPushButton" name="captureButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>560</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>630</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="serialLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>690</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SerialPort:</string>
    </property>
   </widget>
   <widget class="QLabel" name="serialPortStatus">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>690</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Connected!!</string>
    </property>
   </widget>
   <widget class="QLabel" name="camLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera:</string>
    </property>
   </widget>
   <widget class="QLabel" name="camStatusLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>690</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Connected!!</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="speedSpinBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>580</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>560</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>620</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stepps</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="steppsSpinBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>620</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <zorder>horizontalSpinBox</zorder>
   <zorder>quitButton</zorder>
   <zorder>verticalSpinBox</zorder>
   <zorder>graphicsView</zorder>
   <zorder>horizontalLabel</zorder>
   <zorder>verticalLabel</zorder>
   <zorder>takeFrameButton</zorder>
   <zorder>treeWidget</zorder>
   <zorder>compileButton</zorder>
   <zorder>newRowButton</zorder>
   <zorder>verticalSurfaceSpinBox</zorder>
   <zorder>horizontalSurfaceSpinBox</zorder>
   <zorder>cropLabel</zorder>
   <zorder>surfaceLabel</zorder>
   <zorder>captureButton</zorder>
   <zorder>resetButton</zorder>
   <zorder>serialLabel</zorder>
   <zorder>serialPortStatus</zorder>
   <zorder>camLabel</zorder>
   <zorder>camStatusLabel</zorder>
   <zorder>label</zorder>
   <zorder>speedSpinBox</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>steppsSpinBox</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>graphicsView</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>horizontalSpinBox</tabstop>
  <tabstop>verticalSpinBox</tabstop>
  <tabstop>horizontalSurfaceSpinBox</tabstop>
  <tabstop>verticalSurfaceSpinBox</tabstop>
  <tabstop>takeFrameButton</tabstop>
  <tabstop>newRowButton</tabstop>
  <tabstop>compileButton</tabstop>
  <tabstop>quitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
